lib/widget/dropdown.dart

```diff
@@ -96,13 +98,12 @@ abstract class SelectOne extends StatefulWidget with Invokable, HasController<Se
       for (var element in values) {
         // must be of value/label pair. Maybe let user overrides later
         if (element is Map) {
-          if (element['iconname'] != null) {
+          if (element['icon'] != null) {
```

Comment:
no need for If/Else nor checking if element['icon'] is there. You can simply replace it with `SelectOneItem(value, label, icon)`

Reply:
okay